app/game/[gameId]/LiveGame.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import type { Database } from '@/lib/database.types';
+import { useSupabase } from '@/app/supabase-provider';
+import { Turn } from 'mint-works/dist/turn';
+import { RealtimePostgresUpdatePayload } from '@supabase/supabase-js';
+
+export default function LiveGame({
+  gameId,
+  initialGame,
+  playerName,
+}: {
+  gameId: string;
+  initialGame: Database['public']['Tables']['game']['Row'];
```

Comment:
> Also why is deployment failing lol

I believe its because Vercel needs the supabase environment variables, but also the builds fail on linting errors so they need to be resolved or ignored too.